docs/source/linchpincli_linchpin_destroy.rst

```diff
@@ -0,0 +1,35 @@
+linchpin destroy
+================
+
+Usage: ``linchpin destroy [OPTIONS] TARGET``
+
+Destroys nodes from the given target(s) in the given PinFile.
+
+**Arguments**
+
+``TARGET ...``
+	Destroy ONLY the listed target(s). If omitted, ALL targets are destroyed.
```

Comment:
Let's clarify, a user may have multiple PinFiles. Maybe it should read 'If omitted, ALL targets in the appropriate PinFile are destroyed.'